extensions/amp-date-countdown/1.0/amp-date-countdown.js

```diff
@@ -120,6 +120,7 @@ AmpDateCountdown['props'] = {
   'whenEnded': {attr: 'when-ended', type: 'string'},
   'locale': {attr: 'locale', type: 'string'},
   'biggestUnit': {attr: 'biggest-unit', type: 'string'},
+  'countUp': {attr: 'data-count-up', type: 'boolean'},
```

Comment:
Another ask: let's go ahead and remove `data-` from here. I don't see why have it when all other attributes don't.

Reply:
Hmm, so in the original `0.1` change, the components group preferred the `data` prefix primarily because it's the preferred way for adding new attributes to our components w/o overwhelming the namespace.

If we do this and break backwards compatibility we will also need to update the validator to accept both `count-up` and `data-count-up`.  Definitely doable, but I think a little less clean.

Here's the `0.1` change with a bit of context: https://github.com/ampproject/amphtml/pull/31099, but most of the conversation for using `data` took place offline.

Let me know your thoughts! :)

Reply:
`count-up` definitely doesn't overwhelm the namespace. So I'd prefer it for consistency and ergonomics. But also conceptually it's a better naming scheme. Might not be true for all attributes, such as `data-param-x` in social-share, but I believe it's a case here. Let's just mention it in the "Migration notes" of this component.

Reply:
https://github.com/ampproject/amphtml/pull/32201 includes the change to the attribute name.  Thanks for the input here!